### ISSUE TITLE
[FATAL] this import break token plugin

### ISSUE THREAD

**Issue by al3x609** [novnc/websockify#396]
https://github.com/novnc/websockify/blob/70911c6d6ed7d05d2e8ffaba464efa18d045e582/websockify/token_plugins.py#L141-L143

these dependencies are not installed, 
and it is not reported in the manual as a requirement.

**Comment by al3x609** [comment]
```
Traceback (most recent call last):
  File "/mnt/noVNC/websockify/run", line 6, in <module>
    websockify.websocketproxy.websockify_init()
  File "/mnt/noVNC/websockify/websockify/websocketproxy.py", line 696, in websockify_init
    __import__(token_plugin_module)
  File "/mnt/noVNC/websockify/websockify/token_plugins.py", line 141, in <module>
    import redis
ImportError: No module named redis
...
    import simplejson
ImportError: No module named simplejson
```
https://github.com/novnc/websockify/blob/70911c6d6ed7d05d2e8ffaba464efa18d045e582/setup.py#L32
refer to this [ pull reques](https://github.com/novnc/websockify/pull/389)

**Comment by CendioOssman** [comment]
Fixed in 01a184f4a986b5001551e5e89c271b0a78f6c832.

Thank you for reporting this.
